zavod/zavod/exporters/delta.py

```diff
@@ -33,6 +37,15 @@ def generate(self) -> Generator[Any, None, None]:
     def finish(self) -> None:
         with open(self.path, "wb") as fh:
             for op in self.generate():
+                if op["op"] in self.counts:
```

Comment:
Kind of wonder if we don't want the check - if there's an unknown op, let's crash.